### PR TITLE
Update design of household size field

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Update icons in stepper input / household size field
- Make it always 48px height

## Justify why they are needed

Design review.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1739** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1739" target="_blank">Graphite</a> 
    * **PR #1741** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1741" target="_blank">Graphite</a> 
      * **PR #1744** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1744" target="_blank">Graphite</a> 
        * **PR #1745** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1745" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1745?utm_source=stack-comment).

File: apps/store/src/components/PriceCalculator/HouseholdSizeField/HouseholdSizeField.tsx [hunk 1]
```diff
@@ -68,18 +71,24 @@ export const HouseholdSizeField = ({ field, autoFocus = false }: FieldProps) =>
...
-            disabled={value === minValue}
+            disabled={isDecrementDisabled}
           >
-            <Minus />
+            <MinusIcon
+              size="1rem"
+              color={isDecrementDisabled ? theme.colors.textDisabled : theme.colors.textPrimary}
+            />
           </StyledButton>
           <StyledButton
```
Comment (alebedev) [inline apps/store/src/components/PriceCalculator/HouseholdSizeField/HouseholdSizeField.tsx:81]: nit: Maybe we need `IconButton` component to take care of disabled styles

I wonder why Icon inside does not inherit disabled color from button

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/PriceCalculator/HouseholdSizeField/HouseholdSizeField.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/PriceCalculator/HouseholdSizeField/HouseholdSizeField.tsx:81]: good point, that would be a nice refactor

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: HQG0VyZD9uJhmvtmAEMW.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1745)

**Pull request merged**